### PR TITLE
Feature/button screen gallery app

### PR DESCRIPTION
**Pull request by Keith-Go**
## Ticket
- Button Screen: [button screen](https://www.notion.so/42297babdb6541c586c45340529b23f3?v=180009363b23459eb8233bf6052d4758&p=c6365ede1e0c4f2d933fb8b224f47cbc&pm=s)

## Screenshots
image

### CODE REVIEW

File: gallery_app/lib/modules/button/screens/button_screen.dart [hunk 1]
```diff
@@ -1,10 +1,285 @@
...
+          child: const Text(
+            "Option 3",
+          ),
+        )
+      ],
+    );
+  }
+}
+
+class MaterialButtonClip extends CustomClipper<Path> {
```
Comment (luke-nguyen-goldenowl) [inline gallery_app/lib/modules/button/screens/button_screen.dart]: Do not use Material (Flutter's default names), set it according to your custom UI

**Comment by Keith-Go** [comment]
resolved

**Comment by luke-nguyen-goldenowl** [comment]
Material Button and Raw Material Button not correct
Please check design

Screenshot 2023-12-18 at 4 30 05 PM

**Comment by Keith-Go** [comment]
resolved

**Comment by luke-nguyen-goldenowl** [comment]
Approved

**Pull request merged**